backend/pyproject.toml

```
@@ -14,6 +14,8 @@ sqlmodel = "^0.0.22"
 pymysql = "^1.1.1"
 python-dotenv = "^1.0.1"
 pytest = "^8.3.3"
+httpx = "^0.27.2"
+cryptography = "^43.0.3"
 
```

Per tal de calcular la cobertura dels tests sobre el codi afegeix aquesta llibreria.
```suggestion
pytest-cov = "^5.0.0"
```